src/utils/__tests__/addDependenciesToPackages.test.js

```diff
@@ -0,0 +1,77 @@
+// @flow
+
+import { getFixturePath } from 'jest-fixtures';
```

Comment:
This util is not finished yet. I realised I needed to have all of symlinking working first.